common/src/main/java/org/broadleafcommerce/common/i18n/service/DynamicTranslationProvider.java

```diff
@@ -32,18 +32,22 @@
     /**
      * If translations are enabled, this method will look for a translation for the specified field. If translations are
      * disabled or if this particular field did not have a translation, it will return back the defaultValue.
+     * If the passed in locale's language matches the Broadleaf default locale, it will return back the defaultValue.
      * 
      * @param obj
      * @param field
      * @param defaultValue
      * @return the translated value
      */
     public static String getValue(Object obj, String field, final String defaultValue) {
+        BroadleafRequestContext requestContext = BroadleafRequestContext.getBroadleafRequestContext();
         String valueToReturn = defaultValue;
-        
-        if (TranslationConsiderationContext.hasTranslation()) {
+
+        // avoiding translation calls for the default locale
+        if (TranslationConsiderationContext.hasTranslation() &&
+                !requestContext.getLocale().getDefaultFlag()) {
```

Comment:
@Vitalij-Voronkoff On second thought, I think we should refactor this to be `&& localeShouldBeTranslated(requestContext.getLocale())`

Within that method, we can simply return `!locale.getDefaultFlag()`, but it would give users the ability to declare that `en_us` & `en_gb` should not be translated if only `en_us` is the default.

Reply:
... realizing that `getValue(...)` is a static method which really derails the idea of extensibility